src/main/resources/org/jenkinsci/plugins/zap/ZAPDriver/config.jelly

```diff
@@ -128,6 +128,9 @@ SOFTWARE.
             <f:entry title="${%Logged in Indicator}" field="loggedInIndicator">
               <f:textbox clazz="required" />
             </f:entry>
+            <f:entry title="${%Logged out Indicator}" field="loggedOutIndicator">
+              <f:textbox clazz="required" />
```

Comment:
@tlenaic the required is there to indicate that the project will fail if this value is not specified.